data/containers/helm-test/templates/job.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}-job"
```

Comment:
I am thinking that for each deployment in testing we should create a namespace. Then we can destroy the namespace

Reply:
I was thinking actually the same. Let me implement it.

Reply:
Done.